implementation/src/main/java/io/smallrye/jwt/auth/principal/JWTAuthContextInfo.java

```diff
@@ -193,4 +196,12 @@ public String getTokenKeyId() {
     public void setTokenKeyId(String tokenKeyId) {
         this.tokenKeyId = tokenKeyId;
     }
+
+    public Set<String> getExpectedAudience() {
```

Comment:
I have a minor preference to drop `Expected`, even though it is technically a better qualifier, but it just happens that `JWTContentInfo` has not followed this naming pattern for other claims (`iss`, etc). But if you prefer to keep `Expected` then I'll be fine with it, it is up to you.

Reply:
If it's OK, I'd prefer to keep it. Mainly, because it conveys intent at a glance.

Reply:
OK